jujugui/static/gui/src/app/components/user-profile/kpi/test-kpi.js

```diff
@@ -0,0 +1,71 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+var juju = {components: {}}; // eslint-disable-line no-unused-vars
+
+describe('UserProfileEntityKPI', () => {
+  let d3, metrics, metricTypes;
+
+  beforeAll((done) => {
+    YUI().use('user-profile-entity-kpi', () => { done(); });
+  });
+
+  beforeEach(() => {
+    d3 = {};
+    metricTypes = ['metric', 'bad-wolf'];
+    metrics = [
+      {
+        Metric: 'metric',
```

Comment:
I can propagate this:

```
{
  "code": "internal server error",
  "error": "internal server error",
  "message": "internal server error"
}
```

But I take your meaning. I forgot we decided to do that locally and not rely on APIs to do so.